driver-adapters-wasm/pg-cfpages-basic/run.sh

```diff
@@ -7,6 +7,6 @@ pnpm install
 pnpm prisma generate
 
 pnpm wrangler pages functions build --node-compat --outdir build/
-pnpm wrangler publish build/index.js --name pg-cfpages-basic public
+pnpm wrangler publish build/index.js --name pg-cfpages-basic
```

Comment:
Ideally, we would publish it as a page as well?

Reply:
Yeah, this is not working

Reply:
I'll leave this aside for now